lib/std/math/float.zig

```diff
@@ -43,18 +43,18 @@ pub fn floatMantissaBits(comptime T: type) comptime_int {
 }
 
 /// Returns the number of binary digits in the mantissa of floating point type T.
```

Comment:
I feel like this docstring mismatches the new function since it no longer includes the integer part.

Reply:
You're right, good catch. Let me fix that up real quick.